src/Http/Place/UpdateAddressRequestHandler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Place;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\EntityNotFoundException;
+use CultuurNet\UDB3\Http\Deserializer\Address\AddressJSONDeserializer;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Http\Response\NoContentResponse;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Place\Commands\UpdateAddress;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\StringLiteral;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+final class UpdateAddressRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    private DocumentRepository $placeDocumentRepository;
+
+    public function __construct(CommandBus $commandBus, DocumentRepository $placeDocumentRepository)
+    {
+        $this->commandBus = $commandBus;
+        $this->placeDocumentRepository = $placeDocumentRepository;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $placeId = $routeParameters->getPlaceId();
+        $language = $routeParameters->getLanguage();
+
+        $address = (new AddressJSONDeserializer())->deserialize(
+            new StringLiteral($request->getBody()->getContents())
+        );
+
+        try {
+            $this->placeDocumentRepository->fetch($placeId);
+        } catch (DocumentDoesNotExist $e) {
+            throw new EntityNotFoundException(
+                sprintf('Place with id: %s not found.', $placeId)
+            );
```

Comment:
@LucWollants @JonasVHG maybe it would be good to have an improvement ticket (to pick up later) to add a check like this to every event/place endpoint? So we don't have to repeat this in every request handler, but do have this check everywhere. I think it's now missing in a lot of places and it seems very useful to catch mistakes.
And also for other resources maybe?
I can check if it's possible to do this easily in the new router (#1102). Sometimes routers have a feature to look up a wildcard like an id first before the request is sent to a request handler.

Reply:
I created [III-4899](https://jira.uitdatabank.be/browse/III-4899) for this.

Reply:
Ah now that I see the `DocumentDoesNotExist` in the title I just remembered, we already have a solution for this: This exception is caught in the error handler and converted to an API problem. So I think we can just leave out the check in the request handlers. We also have the same for `AggregateNotFoundException`, so if a command is dispatched and the command handler loads the aggregate but gets that exception, we also get a nice API problem right now: https://github.com/cultuurnet/udb3-silex/blob/master/app/Error/WebErrorHandlerProvider.php#L92-L114

So I think we can keep that for now and just remove the check in this request handler (and others that we refactor)?

Reply:
Also `EntityNotFoundException` is not converted to a nice API problem response right now I think, so this extra check in the request handler is currently worse than the error handling that we have. So I would suggest to remove it from this request handler in this PR

Reply:
@JonasVHG the ticket is not needed and the `fetch` with `catch` can be removed.